.devcontainer/Dockerfile

```diff
@@ -80,3 +81,8 @@ ENV TIZEN_ROOTFS /tizen_rootfs
 
 # Fast Model GDB plugins path for debugging support
 ENV FAST_MODEL_PLUGINS_PATH /opt/FastModelsPortfolio_11.16/plugins/Linux64_GCC-9.3
+
+# icecc and ccache for cluster compilation
```

Comment:
I would add better explanation what is going on here, because for someone who is not familiar with ccache and icecc, such setup might not be trivial.

```suggestion
# Set up ccache as a pigweed command launcher when using the scripts/build/build_examples.py
# script. Also, set up icecc as the command prefix for ccache. Such setup allows to benefit
# from compilation caching and distributed compilation at the same time.
#
# NOTE: In order to use distributed compilation with icecc, one should run
#       "scripts/icecc.sh start" before starting the build.
```